src/index.css

```diff
@@ -1,4 +1,338 @@
+}
+
+.inputForm {
+  width: 72vw;
+  height: 4.813rem;
+  margin: 1.313rem 4.85rem 0;
+  padding: 0;
+  padding-bottom: 3rem;
+  border-top: solid 1px #b4b0b0;
+  background-color: #e7e3e3;
+  margin-left: 13%;
+}
+
+.Title {
+  width: 10.875rem;
+  height: 1.5rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.Lesson-Pane2 {
+  width: 15.813rem;
+  height: 2.813rem;
+  border-radius: 4px;
+  border: solid 1px #888;
+  background-color: #fff;
+  margin-right: 2.2rem;
+  padding-left: 0.5rem;
+}
+
+.shadow {
+  width: 85.063rem;
+  height: 63.313rem;
+  opacity: 0.2;
+  border-radius: 4px;
+  -webkit-filter: blur(50px);
+  filter: blur(50px);
+  background-color: var(--black);
+}
+
+.titling {
+  width: 30rem;
+}
+
+.submit {
+  width: 10.5rem;
+  height: 2.813rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: white;
+}
+
+.remove {
+  width: 4.375rem;
+  height: 1.188rem;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  cursor: pointer;
+  border-right: solid 1px #d6d3d3;
+  background-color: white;
+}
+
+.comment {
+  padding-right: 3rem;
+  width: 5rem;
+}
+
+.edit {
+  padding-left: 1rem;
+  width: 3rem;
+  border: none;
+  cursor: pointer;
+  background-color: white;
+}
+
+.author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.category1 {
+  width: auto;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.titleData {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.header {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  padding-top: 1rem;
+  padding-left: 7%;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  white-space: nowrap;
+}
+
+.Booklist {
+  width: 99%;
+}
+
+::-webkit-scrollbar {
+  display: none;
+}
+
+.user {
+  color: #0290ff;
+  font-size: 1.5rem;
+  padding-bottom: 1rem;
+}
```

Comment:
- [ ] Why don't you align the `user` with the cards on the page so as to match the design guideline and also as suggested by the previous reviewer?😄

Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/pGN63TG.png) | ![](https://i.imgur.com/v8MuGAg.png)

Reply:
The form is properly aligned on my own side.

Reply:
**I have one last review left. Please I have implemented all the requirements.**